src/appleseed-max-impl/appleseedvolumemtl/appleseedvolumemtl.cpp

```diff
@@ -5,7 +5,7 @@
 //
 // This software is released under the MIT license.
 //
-// Copyright (c) 2016-2018 Francois Beaune, The appleseedhq Organization
+// Copyright (c) 2016-2018 Sergo Pogosyan, The appleseedhq Organization
```

Comment:
The year needs to be fixed too. Just take the line I was suggesting :)

Reply:
Sorry! I'm too impatient to read the whole comment :)